tests/annotation_test/mod.rs

```diff
@@ -0,0 +1,11 @@
+// Automatically generated by the Flatbuffers compiler. Do not modify.
+mod ab_generated;
+pub use self::ab_generated::*;
+mod any_generated;
+pub use self::any_generated::*;
+mod object_generated;
+pub use self::object_generated::*;
+mod game_generated;
+pub use self::game_generated::*;
+mod annotations_generated;
+pub use self::annotations_generated::*;
```

Comment:
@CasperN  I don't understand that much rust, but it seems that we are making these objects public?